test/integration/annotation/annotated_resource_setup.go

```diff
@@ -0,0 +1,318 @@
+package annotation
+
+import (
+	"context"
+	"fmt"
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/client"
+	"github.com/skupperproject/skupper/pkg/kube"
+	"github.com/skupperproject/skupper/pkg/utils"
+	"github.com/skupperproject/skupper/test/utils/k8s"
+	"gotest.tools/assert"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+	"testing"
+	"time"
+)
+
+const (
+	nsCluster1 = "annotation-1"
+	nsCluster2 = "annotation-2"
+	timeout    = 120 * time.Second
```

Comment:
you can import "github.com/skupperproject/skupper/test/cluster"
and then just use: ImagePullingAndResourceCreationTimeout, in the spirit of centralizing the tuning

Reply:
As per our early discussion, I have raised #209 so we can refactor all integration tests to use a standard
base type that provides these common behaviors.

Reply:
like this you mean?
`timeout    = cluster.ImagePullingAndResourceCreationTimeout`